api/utils/slices_test.go

```diff
@@ -37,3 +38,19 @@ func TestDeduplicate(t *testing.T) {
 		})
 	}
 }
+
+func TestDeduplicateAny(t *testing.T) {
```

Comment:
nit: I'd be nice to also unit test the `Deduplicate()`

Reply:
These tests already exist above here, no change was necessary for the generification.